### DISCUSSION TITLE
1.6.1版解析视频数量

### DISCUSSION THREAD

**Discussion by Whateveryet** [leiurayer/downkyi#981] [Q&A]
1.6.1版最多解析100个视频，然后就不动了。不下载也不继续解析。超过100集的视频也是卡在100集位置就不动了。大家遇到过吗？

**Comment by Whateveryet** [comment]
补充。我是下载这个视频的时候遇到的问题https://www.bilibili.com/video/BV1XR4y1V7da/?spm_id_from=333.337.search-card.all.click&vd_source=4d4621e239c28fa3c13f187e6116a159

**Comment by yaobiao131** [comment]
![image](https://github.com/leiurayer/downkyi/assets/28655758/7c9c5be1-cd35-49ef-8d89-3400864a6292)
修改ua为
![image](https://github.com/leiurayer/downkyi/assets/28655758/d41595e7-287e-4231-a797-1378b64b3b3e)
无问题

**Comment by LcxRzh** [comment]
我的也是，解析视频超过100就卡住了
